### ISSUE TITLE
Download crops the animation

### ISSUE THREAD

**Issue by fgeierst** [piterwilson/squiggle-animation-tool#1]
When I hit the download button (down arrow), a .gif file get's downloaded. But this contains only a cropped version of the cavas I have been drawing onto. It's only the uppler left quarter.

![image](https://user-images.githubusercontent.com/22551442/82824945-8fa7f700-9eaa-11ea-96f5-1a71a87c6e6e.png)

Win10, Edge 81

**Comment by Rampugnant** [comment]
Same issue as @fgeierst. Using the online version. When testing it out, I get the cropped version reliably when the drawing goes off the canvas in any direction. If everything is within the canvas I sometimes get a normal download. Guessing this has something to do with the window being passed to gif.js? Could potentially use the [setOptions method in gif.js](https://github.com/jnordberg/gif.js) to define the width and height. I'm a beginner so, it is just a guess. It would be great to get the site working again. I really enjoy the simplicity of this tool. Thanks for creating it @piterwilson !

**Comment by piterwilson** [comment]
Hey guys, I’m pretty busy these days and might be a while before I can work on this. I am adding a “help wanted” label to this issue hopefully we can get a volunteer to help. Apologies for this issue!

**Comment by Rampugnant** [comment]
@piterwilson I found I could recreate the cropping error on my mac screen (resolution 1792 x 1120), but there were no errors if I just moved the window over to my second desktop (resolution 2560 X 1440) refreshed and tried the download again. Resizing didn't have any effect. 

I found a similar seeming issue listed in gif.js repo - [Setting constructor width & height crops image](https://github.com/jnordberg/gif.js/issues/94). The work around on the comment suggests creating a temporary canvas and stretching the frames to it and then passing it to the worker. 

I'm not confident enough to track down exactly where the handoff to gif.worker.js happens and try to rewrite it. I'd love some help.
